cmake/features.cmake

```diff
@@ -160,33 +160,33 @@ else()
     set(protoc_available ON)
 endif()
 
-ie_dependent_option(NGRAPH_ONNX_FRONTEND_ENABLE "Enable ONNX FrontEnd" ON "protoc_available" OFF)
-ie_dependent_option(NGRAPH_PDPD_FRONTEND_ENABLE "Enable PaddlePaddle FrontEnd" ON "protoc_available" OFF)
-ie_option(NGRAPH_IR_FRONTEND_ENABLE "Enable IR FrontEnd" ON)
-ie_dependent_option(NGRAPH_TF_FRONTEND_ENABLE "Enable TensorFlow FrontEnd" ON "protoc_available" OFF)
-ie_dependent_option(NGRAPH_USE_SYSTEM_PROTOBUF "Use system protobuf" OFF
-    "NGRAPH_ONNX_FRONTEND_ENABLE OR NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_TF_FRONTEND_ENABLE;BUILD_SHARED_LIBS" OFF)
-ie_dependent_option(NGRAPH_UNIT_TEST_ENABLE "Enables ngraph unit tests" ON "ENABLE_TESTS;NOT ANDROID" OFF)
-ie_dependent_option(NGRAPH_UNIT_TEST_BACKENDS_ENABLE "Control the building of unit tests using backends" ON
-    "NGRAPH_UNIT_TEST_ENABLE" OFF)
-ie_option(OPENVINO_DEBUG_ENABLE "Enable output for OPENVINO_DEBUG statements" OFF)
+ie_dependent_option(ENABLE_OV_ONNX_FRONTEND "Enable ONNX FrontEnd" ON "protoc_available" OFF)
+ie_dependent_option(ENABLE_OV_PDPD_FRONTEND "Enable PaddlePaddle FrontEnd" ON "protoc_available" OFF)
+ie_option(ENABLE_OV_IR_FRONTEND "Enable IR FrontEnd" ON)
+ie_dependent_option(ENABLE_OV_TF_FRONTEND "Enable TensorFlow FrontEnd" ON "protoc_available" OFF)
+ie_dependent_option(ENABLE_SYSTEM_PROTOBUF "Use system protobuf" OFF
+    "ENABLE_OV_ONNX_FRONTEND OR ENABLE_OV_PDPD_FRONTEND OR ENABLE_OV_TF_FRONTEND;BUILD_SHARED_LIBS" OFF)
+ie_dependent_option(ENABLE_OV_CORE_UNIT_TESTS "Enables OpenVINO core unit tests" ON "ENABLE_TESTS;NOT ANDROID" OFF)
+ie_dependent_option(ENABLE_OV_CORE_BACKEND_UNIT_TESTS "Control the building of unit tests using backends" ON
+    "ENABLE_OV_CORE_UNIT_TESTS" OFF)
+ie_option(ENABLE_OPENVINO_DEBUG "Enable output for OPENVINO_DEBUG statements" OFF)
```

Comment:
if we just rename option, some private CI maybe incorrect. E.g. private CI disables ONNX FE, after such renaming this `NGRAPH_TF_FRONTEND_ENABLE=OFF` option does nothing.

Can we have aliases for keys like `ie_dependent_option("ENABLE_OV_CORE_BACKEND_UNIT_TESTS;NGRAPH_UNIT_TEST_BACKENDS_ENABLE " "Control the building of unit tests using backends" ON
    "ENABLE_OV_CORE_UNIT_TESTS" OFF)`
Where both keys are valid and synchronized

See how it's done for GPU https://github.com/openvinotoolkit/openvino/blob/88e078460629bc616e5b52b2f03df375e6f52a7b/cmake/features.cmake#L15-L16

Reply:
CC @vinograd47 do you have advices how to achieve this?